ui/src/app/pages/access/module.tsx

```diff
@@ -78,16 +80,19 @@ export const Module = (props: {
       <FlexRow style={styles.moduleCTA}>
         {cond(
           [
-            (clickable || moduleName === AccessModule.ERACOMMONS) &&
+            (active || moduleName === AccessModule.ERACOMMONS) &&
               showRefresh &&
               !!refreshAction,
             () => <Refresh {...{ refreshAction, showSpinner }} />,
           ],
-          [active, () => <Next />]
+          [focused, () => <Next />]
         )}
       </FlexRow>
       <ModuleBox
-        {...{ clickable }}
+        clickable={
+          active &&
+          !(enableRasIdMeLinking && moduleName === AccessModule.IDENTITY)
```

Comment:
The entirety of the IDENTITY module should only be clickable when enableRasIdMeLinking is set to false. That is because the new version of this module has multiple links on it, so a universal onClick functionality would not make sense for this module.